framework/src/service/ServiceReferenceBasePrivate.cpp

```diff
@@ -106,7 +117,7 @@ std::shared_ptr<void> ServiceReferenceBasePrivate::GetService(BundlePrivate* bun
   auto s = ExtractInterface(GetServiceInterfaceMap(bundle), interfaceId);
   if (!s)
   {
-    registration->Lock(), --registration->dependents[bundle];
+    registration->Lock(), --registration->dependents.at(bundle);
```

Comment:
I'm missing something. I don't see a case where ```GetServiceInterfaceMap``` could return a valid ```InterfaceMapConstPtr``` object without ensuring that ```registration->dependents``` has a valid element in it with an incremented count.